core/backend/src/ImageSourceConversion.ts

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
```

Comment:
Believe you'll need a 
```
/** @packageDocumentation
 * @module {Group}
 */
```

Reply:
Thanks, couldn't find a suitable group, might have to make a new one.